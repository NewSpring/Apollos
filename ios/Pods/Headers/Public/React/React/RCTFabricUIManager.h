../../../../../../node_modules/react-native/React/Fabric/RCTFabricUIManager.h