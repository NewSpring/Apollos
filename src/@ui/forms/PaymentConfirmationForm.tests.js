import React from 'react';
import renderer from 'react-test-renderer';
import moment from 'moment';
import Providers from '@ui/TestProviders';

import { PaymentConfirmationFormWithoutData } from './PaymentConfirmationForm';

const createTestData = () => ({
  isLoading: false,
  campus: 'web',
  contributions: {
    frequencyId: 'today',
    startDate: moment('10/20/2120').toDate(),
    contributions: [{
      name: 'General',
      amount: 253,
    }],
    isPaying: false,
  },
  onSubmit: jest.fn(),
});

describe('The PaymentConfirmationForm component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Providers>
        <PaymentConfirmationFormWithoutData {...createTestData()} />
      </Providers>,
    );
    expect(tree).toMatchSnapshot();
  });
  it('renders a loading indicator', () => {
    const data = createTestData();
    data.isLoading = true;
    const tree = renderer.create(
      <Providers>
        <PaymentConfirmationFormWithoutData {...data} />
      </Providers>,
    );
    expect(tree).toMatchSnapshot();
  });
  it('shows a paying state', () => {
    const data = createTestData();
    data.contributions.isPaying = true;
    const tree = renderer.create(
      <Providers>
        <PaymentConfirmationFormWithoutData {...data} />
      </Providers>,
    );
    expect(tree).toMatchSnapshot();
  });
  it('shows a scheduled contribution', () => {
    const data = createTestData();
    data.contributions.frequencyId = 'monthly';
    const tree = renderer.create(
      <Providers>
        <PaymentConfirmationFormWithoutData {...data} />
      </Providers>,
    );
    expect(tree).toMatchSnapshot();
  });
});
