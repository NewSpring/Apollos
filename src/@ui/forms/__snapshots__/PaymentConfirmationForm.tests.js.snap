// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The PaymentConfirmationForm component renders a loading indicator 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
    style={
      Object {
        "alignSelf": "center",
        "bottom": 0,
        "flex": 1,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;

exports[`The PaymentConfirmationForm component renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderBottomWidth": 0.5,
        "borderColor": "rgba(133, 133, 133, 0.30000000000000004)",
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "borderTopWidth": 0.5,
        "marginBottom": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#858585",
            "fontFamily": "Colfax-Regular",
            "fontSize": 14,
            "lineHeight": 15.78,
            "paddingBottom": 1.8,
            "paddingLeft": 0,
            "paddingTop": 3.6,
          }
        }
      >
        Campus: 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#505050",
            "fontFamily": "Colfax-Bold",
            "fontSize": 15.75,
            "lineHeight": 17.75,
            "paddingBottom": 1.8,
            "paddingTop": 4.5,
          }
        }
      >
        web
      </Text>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 0,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        General
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          $
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 20.02,
              "lineHeight": 25.33,
              "paddingBottom": 3.15,
              "paddingTop": 4.5,
            }
          }
        >
          253
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          .00
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 0,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        Total
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          $
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 25.2,
              "lineHeight": 31.28,
              "paddingBottom": 3.71,
              "paddingTop": 5.04,
            }
          }
        >
          253
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          .00
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 20,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#303030",
          "fontFamily": "Colfax-Regular",
          "fontSize": 14,
          "lineHeight": 15.78,
          "paddingBottom": 1.8,
          "paddingTop": 3.6,
        }
      }
    >
      To finish giving, you'll be redirected to Safari to securely complete this contribution.
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 0,
      }
    }
    vertical={false}
  >
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        bordered={false}
        disabled={false}
        pill={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#6bac43",
            "borderColor": "#6bac43",
            "borderRadius": 60,
            "borderWidth": 2,
            "elevation": 2,
            "flexDirection": "row",
            "height": 60,
            "justifyContent": "center",
            "opacity": 1,
            "overflow": "hidden",
            "paddingHorizontal": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#ffffff",
              "fontFamily": "Colfax-Bold",
              "fontSize": 20.02,
              "lineHeight": 25.33,
              "paddingBottom": 3.15,
              "paddingTop": 4.5,
            }
          }
        >
          Submit
           
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#6bac43",
            "padding": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Change Payment Method
      </Text>
    </View>
  </View>
</View>
`;

exports[`The PaymentConfirmationForm component shows a paying state 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderBottomWidth": 0.5,
        "borderColor": "rgba(133, 133, 133, 0.30000000000000004)",
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "borderTopWidth": 0.5,
        "marginBottom": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#858585",
            "fontFamily": "Colfax-Regular",
            "fontSize": 14,
            "lineHeight": 15.78,
            "paddingBottom": 1.8,
            "paddingLeft": 0,
            "paddingTop": 3.6,
          }
        }
      >
        Campus: 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#505050",
            "fontFamily": "Colfax-Bold",
            "fontSize": 15.75,
            "lineHeight": 17.75,
            "paddingBottom": 1.8,
            "paddingTop": 4.5,
          }
        }
      >
        web
      </Text>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 0,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        General
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          $
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 20.02,
              "lineHeight": 25.33,
              "paddingBottom": 3.15,
              "paddingTop": 4.5,
            }
          }
        >
          253
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          .00
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 0,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        Total
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          $
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 25.2,
              "lineHeight": 31.28,
              "paddingBottom": 3.71,
              "paddingTop": 5.04,
            }
          }
        >
          253
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          .00
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 20,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#303030",
          "fontFamily": "Colfax-Regular",
          "fontSize": 14,
          "lineHeight": 15.78,
          "paddingBottom": 1.8,
          "paddingTop": 3.6,
        }
      }
    >
      To finish giving, you'll be redirected to Safari to securely complete this contribution.
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 0,
      }
    }
    vertical={false}
  >
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
          "disabled",
        ]
      }
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        bordered={false}
        disabled={false}
        pill={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#6bac43",
            "borderColor": "#6bac43",
            "borderRadius": 60,
            "borderWidth": 2,
            "elevation": 2,
            "flexDirection": "row",
            "height": 60,
            "justifyContent": "center",
            "opacity": 1,
            "overflow": "hidden",
            "paddingHorizontal": 20,
          }
        }
      >
        <ActivityIndicator
          animating={true}
          color="#ffffff"
          hidesWhenStopped={true}
          size="small"
        />
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#6bac43",
            "padding": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Change Payment Method
      </Text>
    </View>
  </View>
</View>
`;

exports[`The PaymentConfirmationForm component shows a scheduled contribution 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderBottomWidth": 0.5,
        "borderColor": "rgba(133, 133, 133, 0.30000000000000004)",
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "borderTopWidth": 0.5,
        "marginBottom": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#858585",
            "fontFamily": "Colfax-Regular",
            "fontSize": 14,
            "lineHeight": 15.78,
            "paddingBottom": 1.8,
            "paddingLeft": 0,
            "paddingTop": 3.6,
          }
        }
      >
        Campus: 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#505050",
            "fontFamily": "Colfax-Bold",
            "fontSize": 15.75,
            "lineHeight": 17.75,
            "paddingBottom": 1.8,
            "paddingTop": 4.5,
          }
        }
      >
        web
      </Text>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 0,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        General
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          $
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 20.02,
              "lineHeight": 25.33,
              "paddingBottom": 3.15,
              "paddingTop": 4.5,
            }
          }
        >
          253
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          .00
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      horizontal={false}
      style={
        Object {
          "paddingHorizontal": 0,
          "paddingVertical": 20,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        Schedule Details
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "flex-start",
            "paddingHorizontal": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#858585",
              "fontFamily": "Colfax-Regular",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          Frequency: 
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#505050",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          once a month
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "flex-start",
            "paddingHorizontal": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#858585",
              "fontFamily": "Colfax-Regular",
              "fontSize": 14,
              "lineHeight": 15.78,
              "paddingBottom": 1.8,
              "paddingTop": 3.6,
            }
          }
        >
          Starting: 
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#505050",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          10/20/2120
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "height": 0.5,
          "marginLeft": 13.333333333333334,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "flex-start",
          "padding": 10,
          "paddingLeft": 13.333333333333334,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#303030",
            "flexGrow": 1,
            "flexShrink": 1,
            "fontFamily": "Colfax-Bold",
            "fontSize": 20.02,
            "lineHeight": 25.33,
            "paddingBottom": 3.15,
            "paddingLeft": 0,
            "paddingRight": 10,
            "paddingTop": 4.5,
          }
        }
      >
        Total
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          $
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 25.2,
              "lineHeight": 31.28,
              "paddingBottom": 3.71,
              "paddingTop": 5.04,
            }
          }
        >
          253
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#303030",
              "fontFamily": "Colfax-Bold",
              "fontSize": 15.75,
              "lineHeight": 17.75,
              "paddingBottom": 1.8,
              "paddingTop": 4.5,
            }
          }
        >
          .00
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 20,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#303030",
          "fontFamily": "Colfax-Regular",
          "fontSize": 14,
          "lineHeight": 15.78,
          "paddingBottom": 1.8,
          "paddingTop": 3.6,
        }
      }
    >
      To finish giving, you'll be redirected to Safari to securely complete this contribution.
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 0,
      }
    }
    vertical={false}
  >
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        bordered={false}
        disabled={false}
        pill={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#6bac43",
            "borderColor": "#6bac43",
            "borderRadius": 60,
            "borderWidth": 2,
            "elevation": 2,
            "flexDirection": "row",
            "height": 60,
            "justifyContent": "center",
            "opacity": 1,
            "overflow": "hidden",
            "paddingHorizontal": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#ffffff",
              "fontFamily": "Colfax-Bold",
              "fontSize": 20.02,
              "lineHeight": 25.33,
              "paddingBottom": 3.15,
              "paddingTop": 4.5,
            }
          }
        >
          Submit
           
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#6bac43",
            "padding": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Change Payment Method
      </Text>
    </View>
  </View>
</View>
`;
